sdk/wspb/wspb.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+// Package wspb provides convenience functions for reading and writing protobuf
+// messages via a websocket.
+package wspb
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/hashicorp/boundary/internal/errors"
```

Comment:
Probably need to switch to just using the stdlib `errors` package instead.

Reply:
yes, thanks